src/states/elder/mod.rs

```diff
@@ -2045,6 +1942,17 @@ impl Approved for Elder {
         self.pfx_is_successfully_polled
     }
 
+    fn handle_accumulated_message(&mut self, digest: Digest256) -> Result<(), RoutingError> {
+        let routing_message =
+            if let Some(routing_message) = self.signed_messages_accumulator.get(&digest) {
+                routing_message.clone()
+            } else {
+                return Ok(());
+            };
+        self.msg_queue.push_back(routing_message);
```

Comment:
Are we sure that pushing a msg to the msg_queue is what we want? I'm struggling to convince myself of the correctness because it's quite different from the previous code, so I'm not mapping it properly in my head...

Reply:
It also took me some time to catch up with the changed behaviour.  :(
At least according to the test, this pushing to the msg_queue allowed nodes joining network properly for certain steps.